nyc_data/nyc_data/urls.py

```diff
@@ -22,7 +22,8 @@ def trigger_error(request):
     division_by_zero = 1 / 0
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path("admax/", admin.site.urls),
```

Comment:
probably need readme instructions about `createsuperuser` for local dev (Or if there's a way to create a user automatically on local envs?)

Reply:
For future reference, this is a good resource that explained things pretty well re: `createsuperuser`: https://docs.djangoproject.com/en/1.8/intro/tutorial02/